app/changes/team_seed-export

```diff
@@ -0,0 +1 @@
+[Added] [#4233](https://github.com/cosmos/lunie/issues/4233) Seed export @mariopino
```

Comment:
another usurper! xD it should say team 😉

Reply:
Ops, sorry!!! I forgot!